src/github.com/getlantern/buuid/buuid.go

```diff
@@ -58,7 +62,9 @@ func (id ID) Write(b []byte) error {
 // ToBytes returns a 16-byte representation of this ID
 func (id ID) ToBytes() []byte {
 	b := make([]byte, EncodedLength)
-	id.Write(b)
+	if err := id.Write(b); err != nil {
+		log.Debugf("Unable to write as 16-bytes representation: %v", err)
```

Comment:
I think think should be an Error too --- unlikely to ever happen, but if it does it's concerning I think.